### PR TITLE
`liveReloadBaseUrl` is Not Respected

### PR DESCRIPTION
**Pull request by jherdman**
Failing test for #68

### CODE REVIEW

File: tests/inject-live-reload-test.js [hunk 1]
```diff
@@ -95,6 +95,15 @@ describe('dynamicScript returns right script when', hooks => {
     assert.strictEqual(actual, 'http://localhost:35729/_lr/livereload.js?port=35729&host=localhost&path=_lr/livereload');
   });
 
+  it('liveReloadBaseUrl is provided', assert => {
```
Comment (rwjblue) [inline tests/inject-live-reload-test.js]: Can you append `GH#68` to the test name (so we can more easily remember the original issue)?

**Review by rwjblue — COMMENTED** [review]
Haven't had a chance to clone this yet and GH is not sending webhook notifications (so Travis won't kick off), what is the actual value at the moment?

File: tests/inject-live-reload-test.js (hunk 1 above)
Comment (jherdman) [reply tests/inject-live-reload-test.js]: Done!

**Comment by jherdman** [comment]
```
not ok 5 dynamicScript returns right script when > liveReloadBaseUrl is provided
  ---
  message: "failed"
  severity: failed
  actual: "/_lr/livereload.js?port=4200&host=localhost&path=_lr/livereload"
  expected: "http://berus.com:4206/_lr/livereload.js?port=4206&host=berus.com"
  stack:     at Object.assert (/Users/james/Projects/ember-cli-inject-live-reload/tests/inject-live-reload-test.js:104:12)
  ...
```

File: tests/inject-live-reload-test.js [hunk 2]
```diff
@@ -95,7 +95,16 @@ describe('dynamicScript returns right script when', hooks => {
...
+    options.liveReloadBaseUrl = 'http://berus.com:4206';
+
+    let script = InjectLiveReload.dynamicScript(options);
+    let actual = getScriptSrc(script);
+
+    assert.strictEqual(actual, 'http://berus.com:4206/_lr/livereload.js?port=4206&host=berus.com');
+  });
+
+  it('GH#68 liveReloadPrefix is provided', assert => {
```
Comment (rwjblue) [inline tests/inject-live-reload-test.js]: Hmm, seems like you updated the wrong test name?

File: tests/inject-live-reload-test.js [hunk 3]
```diff
@@ -95,7 +95,16 @@ describe('dynamicScript returns right script when', hooks => {
     assert.strictEqual(actual, 'http://localhost:35729/_lr/livereload.js?port=35729&host=localhost&path=_lr/livereload');
   });
 
-  it('liveReloadPrefix is provided', assert => {
+  it('liveReloadBaseUrl is provided', assert => {
```
Comment (rwjblue) [inline tests/inject-live-reload-test.js]: Can you make this one `liveReloadBaseURL including port is provided GH#68`?

Comment (jherdman) [reply tests/inject-live-reload-test.js]: Done! Serves me right for trying to multi-task...

**Comment by erkie** [comment]
@jherdman @rwjblue sorry to bump and old thread, but is this bug waiting on this PR to be merged? In that case, can I jump in somehow and fix the test so it can be looked at?

**Comment by jherdman** [comment]
Hey gang. Yeah, this still needs to be handled and is currently blocking for me.

**Pull request closed without merging**